### DISCUSSION TITLE
Getting error when run 'npm-run-dev'

### DISCUSSION THREAD

**Discussion by sunilsingh2019** [vercel/next.js#40460] [Help]
./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[1]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[2]!./styles/index.css
Error: Cannot find module 'caniuse-lite/data/features/css-unicode-bidi'
Require stack:
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/autoprefixer/data/prefixes.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/autoprefixer/lib/autoprefixer.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/next/dist/build/webpack/config/blocks/css/index.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/next/dist/build/webpack/config/index.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/next/dist/build/webpack-config.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/next/dist/server/dev/hot-reloader.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/next/dist/server/dev/next-dev-server.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/next/dist/server/next.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/next/dist/server/lib/start-server.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/next/dist/cli/next-dev.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/next/dist/lib/commands.js
- /Users/Webee/Desktop/cms-wordpress-app/node_modules/next/dist/bin/next

**Comment by icyJoseph** [comment]
Hi, 

There was a bug in a 3rd party dependency, which has been fixed a couple of hours ago: https://github.com/vercel/next.js/issues/40461#issuecomment-1243306741

Please remove `node_modules` and the `.next` folder, if any, reinstall, and try again!

**Reply by LucaMaia** [reply]
Fiz este procedimento e ainda continua o mesmo erro , sabe o que posso fazer?

**Reply by icyJoseph** [reply]
`rm -rf node_modules .next && npm install`

Remove old `node_modules`, and the pre-compiled stuff in `.next`.

Maybe the bug is back again... 😢 I can take a look later.
